components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/dao/impl/OrganizationManagementDAOImpl.java

```diff
@@ -960,6 +962,23 @@ public String resolveOrganizationId(String tenantDomain) throws OrganizationMana
         }
     }
 
+    @Override
+    public List<String> getAncestorOrganizationIds(String organizationId) throws OrganizationManagementServerException {
+
+        NamedJdbcTemplate namedJdbcTemplate = Utils.getNewTemplate();
+        List<String> ancestorOrganizationIds;
+        try {
+            ancestorOrganizationIds = namedJdbcTemplate.executeQuery(GET_ANCESTORS_OF_GIVEN_ORG,
+                    (resultSet, rowNumber) -> resultSet.getString(1),
+                    namedPreparedStatement -> {
+                        namedPreparedStatement.setString(DB_SCHEMA_COLUMN_NAME_ID, organizationId);
+                    });
+        } catch (DataAccessException e) {
+            throw handleServerException(ERROR_CODE_ERROR_WHILE_RETRIEVING_ANCESTORS, e, organizationId);
+        }
+        return ancestorOrganizationIds;
+    }
```

Comment:
ancestorOrganizationIds is kind of redundant
```suggestion
        try {
            return namedJdbcTemplate.executeQuery(GET_ANCESTORS_OF_GIVEN_ORG,
                    (resultSet, rowNumber) -> resultSet.getString(1),
                    namedPreparedStatement -> {
                        namedPreparedStatement.setString(DB_SCHEMA_COLUMN_NAME_ID, organizationId);
                    });
        } catch (DataAccessException e) {
            throw handleServerException(ERROR_CODE_ERROR_WHILE_RETRIEVING_ANCESTORS, e, organizationId);
        }
    }
```

Reply:
fix in [9de48e4](https://github.com/wso2-extensions/identity-organization-management/pull/90/commits/9de48e40443fc6c48ce4e9f34c0b06b57ff8129c)